server/lib/report_server/reports/tree.ex

```diff
@@ -124,10 +125,12 @@ defmodule ReportServer.Reports.Tree do
         })
       ]},
       %ReportGroup{slug: "student-reports", title: "Student Reports", subtitle: "Reports about students", tbd: true, children: [
-        TBDReport.new(%Report{
+        StudentActions.new(%Report{
```

Comment:
Should we make the "student-reports" Group no longer `:tbd` now?

Reply:
Yes - I'll change that in my next PR.